tidysic/ordering.py

```diff
@@ -0,0 +1,24 @@
+from tidysic.tag import Tag
+from tidysic.formatted_string import FormattedString
+
+
+class OrderingStep:
```

Comment:
This should be either a `@dataclass` or a `NamedTuple`.

I would probably go with `NamedTuple` for the immutability.

(some ref: https://stackoverflow.com/questions/51671699/data-classes-vs-typing-namedtuple-primary-use-cases)